### PR TITLE
Add new Finnish language packages

### PR DESCRIPTION
**Pull request by mahtiankka**
I have translated using the poedit program. Tested in Mint 21.3.

### CODE REVIEW

File: move-into-a-new-folder@anaximeno/files/move-into-a-new-folder@anaximeno/po/fi.po [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+msgid "New Folder"
+msgstr "Uusi kansio"
+
+#: text.py:20
+#, python-format
+msgid ""
+"Folder '%s' already exists inside the current directory, do you want to move "
+"the selected files inside?"
+msgstr ""
+"Kansio \"%s\" on jo hakemistossa. Haluatko siirtää valitut tiedostot sisään?"
```
Comment (rcalixte) [inline move-into-a-new-folder@anaximeno/files/move-into-a-new-folder@anaximeno/po/fi.po:40]: Just a note that there are a few instances of double quotes where the original uses single quotes. This is why the escaping happens in the translation.

Comment (mahtiankka) [reply move-into-a-new-folder@anaximeno/files/move-into-a-new-folder@anaximeno/po/fi.po:40]: Help! Do I have to solve that " " thing somehow?

Comment (rcalixte) [reply move-into-a-new-folder@anaximeno/files/move-into-a-new-folder@anaximeno/po/fi.po:40]: Poedit should let you change the double quotes to single or I can push a change for each if you would like.

Comment (mahtiankka) [reply move-into-a-new-folder@anaximeno/files/move-into-a-new-folder@anaximeno/po/fi.po]: I made the change using the github editor. Apparently it worked? Let me know if it didn't work. Thanks for letting me know - I wouldn't have noticed that myself.

Comment (rcalixte) [reply move-into-a-new-folder@anaximeno/files/move-into-a-new-folder@anaximeno/po/fi.po]: I'll take a look and then merge. Feel free to push any other edits you have in the meanwhile!

**Review by rcalixte — APPROVED** [review]

**Pull request merged**